modules/apps/server/server-admin-web/src/main/resources/META-INF/resources/script.jsp

```diff
@@ -61,6 +65,8 @@ String scriptOutput = (String)SessionMessages.get(renderRequest, "scriptOutput")
 
 		<aui:input cssClass="lfr-textarea-container" name="script" resizable="<%= true %>" type="textarea" value="<%= script %>" />
 
+		<liferay-captcha:captcha />
```

Comment:
For here, based on https://liferay.atlassian.net/browse/LPD-1558?focusedCommentId=2553257 and also our diucssion on slack before "Also, another thing to consider is whether we should continue the mentioned captcha pattern, where we provide users a way to enable/disable this captcha. For instance, if the security team expects that it always be displayed, does that mean we shouldn't provide the ability to disable the captcha?"

I think it will be a better option to make this captcha enabled all the time because:
1. in this case we can make sure we are safe from security bug
2. we do not need new API to check enable, and it is easier to backport.